docs/platforms.md

```diff
@@ -121,5 +121,6 @@ type AndroidDependencyConfig = {
   libraryName?: string | null;
   componentDescriptors?: string[] | null;
   androidMkPath?: string | null;
+  cmakePath?: string | null;
```

Comment:
Do you think `cmakeListsPath` would make more sense due to being more explicit? I'm good with both, but have a slight favor for explicitness